packages/provider/src/MultiChainProvider.ts

```diff
@@ -126,11 +126,17 @@ export class MultiChainProvider extends SafeEventEmitter implements Provider {
 
     assertIsMultiChainRequest(args);
 
+    // We're doing it this way to avoid sentRequest.params = undefined.
+    const sentRequest: Json = { method: args.request.method };
+    if (args.request.params !== undefined) {
+      sentRequest.params = args.request.params;
+    }
```

Comment:
See comment above.

Reply:
Negative, this is user provided request. `{ method: args.request.method , params: args.request.params ?? null }` can result in `{ method: 'asd', params: null }` which is a different object than the user intended, which should be `{ method: 'asd' }`. The null gets serialized into JSON.